modules/getting-started-openshift-common-terms.adoc

```diff
@@ -0,0 +1,52 @@
+
+Deployment and DeploymentConfig::
+{product-title} implements both Kubernetes deployments and {product-title} DeploymentConfigs. Users may select either.
++
+Deployments control how an application is rolled out as pods. They identify the name of the container image to be taken from the registry and deployed as a pod on a node. They set the number of replicas of the pod to deploy, creating a ReplicaSet to manage the process. The labels indicated instruct the scheduler onto which nodes to deploy the pod. The set of labels is included in the pod definition that the ReplicaSet instantiates.
++
+Deployments are able to update the pods deployed onto the worker nodes based on the version of the deployment and the various roll-out strategies for managing acceptable application availability. {product-title} DeploymentConfigs add the additional features of change triggers, which are able to automatically create new versions of the deployment as new versions of the container image are available, or other changes.
+
+Operators::
+An Operator is a Kubernetes-native application. The goal of an Operator is to put operational knowledge into software. Previously this knowledge only resided in the minds of administrators, various combinations or shell scripts or automation software such as Ansible. It was outside your Kubernetes cluster and hard to integrate. With operators, all of this changes.
++
+Operators are purpose-built for your applications. They implement and automate common Day 1 activities such as installation and configuration as well as Day 2 activities such as scaling up and down, reconfiguration, updates, backups, fail overs, and restores in a piece of software running inside your Kubernetes cluster by integrating natively with Kubernetes concepts and APIs. This is called a Kubernetes-native application.
++
+With Operators, you have to treat an application as a collection of primitives such as pods, deployments, services, or ConfigMaps. Instead you can treat it as a single object that exposes only the options that make sense for the application.
+
+Pod::
+A pod is one or more containers deployed together on one host. It consists of a colocated group of containers with shared resources such as volumes and IP addresses. A pod is also the smallest compute unit defined, deployed, and managed.
++
+In {product-title}, pods replace individual application containers as the smallest deployable unit.
++
+Pods are the orchestrated unit in {product-title}. {product-title} schedules and runs all containers in a pod on the same node. Complex applications are made up of many pods, each with their own containers. They interact externally and also with another inside the {product-title} environment. {product-title} runs container images in containers wrapped by a meta object called "pod".  It is possible to have multiple containers in a single pod. For example, to support cluster features you can have a "sidecar" container.
+
```

Comment:
Does it makes sense to move the Pod definition before the Deployment and DeploymentConfig section? The definitions are currently alphabetical, but the Deployment section refers to pods, and it might help to have the Pod definition before deployment.

Reply:
I agree, putting it in a hierarchical order is more intuitive, I would put it in the following order:

Container
POD
ReplicaSet and ReplicationController
Deployment and DeploymentConfig
Service
Route (if added)
Builds
Projects
Operators